flink-formats/flink-json/src/main/java/org/apache/flink/formats/json/JsonRowDeserializationSchema.java

```diff
@@ -351,76 +377,131 @@ private DeserializationRuntimeConverter createFallbackConverter(Class<?> valueTy
 		}
 	}
 
+	private float convertToFloat(ObjectMapper mapper, JsonNode jsonNode) {
+		try {
+			return Float.parseFloat(jsonNode.asText().trim());
+		} catch (NumberFormatException e) {
+			throw createParseErrorException("Unable to deserialize float.", e);
```

Comment:
If this is a parser exception, then we can just ignore exception here instead of propagating it, this will also be better for performance. 

```java
} catch (NumberFormatException e) {
  if (!ignoreParseErrors) {
     throw e;
  }
}
```

Reply:
What result should we return if we just ignore the parse exception here? It seems that throwing an exception may be a good choice. What do you think? @wuchong

Reply:
Yes. We should return null if errors is ignored. We can wrap this logic into a method to reduce duplicate code.

Reply:
Just to confirm, we return null fields for invalid fields and return null rows for invalid json string when 'ignoreParseErrors' is 'ture'. Am I right?

Reply:
If the whole json string is invalid, we can simply drop this record, instead emit a null Row. But for fields, we should use `null` if is invalid.

Reply:
We could not drop the record for invalid json string, as there must be a return value for JsonRowDeserializationSchema.deserialize.

Reply:
Yes, you are right. That will be resolved in new DeserializationSchema in the future. Let's return `null` for now.